addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,422 @@
+/* Full page */
```

Comment:
addon should remove labels in the sounds tab as it does for the costumes tab
![image](https://user-images.githubusercontent.com/72760579/195226262-50efc883-8254-4ed0-bed3-e35f1357bd7e.png)